app/metrics.yaml

```diff
@@ -5840,3 +5786,52 @@ start_on_home:
     notification_emails:
       - android-probes@mozilla.com
     expires: "2022-06-16"
+credit_cards:
+  manual_save_card:
+    type: counter
+    description: |
+      A counter of the number of credit cards that have been saved
+      manually by the user.
+    send_in_pings:
+      - metrics
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/18711
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/19548#issuecomment-848811030
```

Comment:
I think we want to keep the 2 data_reviews that are listed prior.

Reply:
This is the original data review. Not sure what other data reviews you are referring to.